.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
 stages:
   - name: test
   - name: release
-    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
+    if: ((branch = master) OR (tag IS present)) AND NOT fork
```

Comment:
Can you please explain this?

Reply:
I'm hoping that this will publish from master even when not on a tag, i.e. a snapshot

Reply:
I'll have to check this, my understanding is very limited however I think snapshots are already published.

Reply:
you're right! https://oss.sonatype.org/content/repositories/snapshots/com/avast/scala-server-toolkit-http4s-blaze-server_2.12/